docs/components/connectors/introduction.md

```diff
@@ -26,12 +26,13 @@ A Connector consists of two parts:
 
 Users interacting with Connectors may only need to understand the configuration options exposed by the Connector Template in the properties panel.
 
-Connectors are available in three categories - [out-of-the-box (OOTB)](./out-of-the-box-connectors/available-connectors-overview.md) , [Connector Templates](manage-connector-templates.md), and custom via the [Connector SDK](./custom-built-connectors/connector-sdk.md).
+Connectors are available in three categories - [out-of-the-box (OOTB)](./out-of-the-box-connectors/available-connectors-overview.md), [Connector Templates](manage-connector-templates.md), and custom via the [Connector SDK](./custom-built-connectors/connector-sdk.md).
```

Comment:
I am not sure whether this categorization is really clear as the OOTB connectors come with Connector templates as well.

Reply:
Reworded in latest commit to:

Connectors are available [out-of-the-box (OOTB)](./out-of-the-box-connectors/available-connectors-overview.md) and come with [Connector Templates](manage-connector-templates.md) which customize how a BPMN element is shown,
and how it can be configured by process developers. Connector templates are a specific kind of [element templates](/components/modeler/desktop-modeler/element-templates/about-templates.md), which can also be used when creating custom Connectors via the [Connector SDK](./custom-built-connectors/connector-sdk.md).

WDYT?